tests/aggregation/tests.py

```diff
@@ -2158,6 +2177,18 @@ def test_having_with_no_group_by(self):
         )
         self.assertEqual(list(author_qs), [337])
 
+    def test_string_agg(self):
+        values = Publisher.objects.aggregate(
+            stringagg=StringAgg("name", delimiter=";")
```

Comment:
```suggestion
            stringagg=StringAgg("name", delimiter=Value(";"))
```

It'd be great to add testing for expression delimiters as well!